src/queries/utils/index.js

```diff
@@ -0,0 +1,15 @@
+import r from 'rethinkdb';
+
+export const convertId = (id) => ({
```

Comment:
Even though this is a basic utility, the name is a bit vague. Maybe, `convertIdToResourcePointer`? If we're creating Redink-specific vocabulary, we might as well embrace it. Considering this, `convertManyIds` should probably be called `convertIdsToResourcePointers`.